### ISSUE TITLE
Installation error about conflict dependencies on Pandas with idmtools and datar

### ISSUE THREAD

**Issue by YeChen-IDM** [InstituteforDiseaseModeling/malaria-model_validation#27]
Error about conflict dependencies when installing our package:

```
ERROR: Cannot install idmtools-cli and malaria-model-validation because these package versions have conflicting dependencies.
The conflict is caused by:
    datar 0.0.0 depends on pandas<2.0 and >=1.2
    plotnine 0.1.0 depends on pandas>=0.19.0
    idmtools 1.6.6 depends on pandas<1.2 and >=1.1.4
```
Patch release request sent to idmtools team:
https://github.com/InstituteforDiseaseModeling/idmtools/issues/1774
